.github/actions/setup-node-and-pnpm/action.yml

```diff
@@ -17,7 +17,7 @@ runs:
       uses: pnpm/action-setup@v2
       with:
         run_install: false
-        version: 9
+        version: 10.16.0
```

Comment:
There's been a new `pnpm` release `v10.17.0` which adds slightly more functionality to the `minimumReleaseAgeExclude` setting. On the off chance that we use that in the future, let's add that minor bump with this major bump, since it has no extra migration cost.

This release came out very recently (less than 60 hours ago), so I missed it initially too.